src/sentry/integrations/vercel/integration.py

```diff
@@ -142,18 +142,25 @@ def get_slug(self):
             user = client.get_user()
             return user["username"]
 
-    def get_organization_config(self):
-        vercel_client = self.get_client()
-        # TODO: add try/catch if we get API failure
-        slug = self.get_slug()
+    def _get_vercel_projects(self):
+        """If the API returns a 403, just silently replace the slug."""
+        try:
+            slug = self.get_slug()
+        except ApiError:
+            slug = ""
+
         base_url = "https://vercel.com/%s" % slug
-        vercel_projects = [
+
+        client = self.get_client()
+        return [
             {"value": p["id"], "label": p["name"], "url": "{}/{}".format(base_url, p["name"])}
-            for p in vercel_client.get_projects()
+            for p in client.get_projects()
```

Comment:
@mgaeta aren't these links going to be bogus if we use an empty string for the slug?